Tasks/ServiceFabricDeploy/utilities.ps1

```diff
@@ -146,8 +146,8 @@ function Read-PublishProfile
     $publishProfileElement = $publishProfileXml.PublishProfile
     $publishProfile = @{}
 
-    $publishProfile.ClusterConnectionParameters = Read-XmlElementAsHashtable $publishProfileElement.Item("ClusterConnectionParameters")
```

Comment:
The VSTS task makes use of a service endpoint which provides all the necessary information we need to make the connection (credentials, certificate, etc), which the publish profile doesn't have.  Since service endpoints are the natural mechanism for describing such things in VSTS, we make use of that instead of relying on the publish profile for that information.